apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/NormalizedCacheStore.java

```diff
@@ -0,0 +1,17 @@
+package com.apollographql.android.cache.normalized;
+
+import java.util.Collection;
+
+public abstract class NormalizedCacheStore {
```

Comment:
Curious to your architecture opinion on this. I tend to lean towards requiring implementations to catch their own `IO` related errors. 
Generally, an `exception` on read can be replaced with null.

I wouldn't be sure what to do with `exception` on a merge though..